Sources/SmileID/Classes/SmileID.swift

```diff
@@ -43,37 +46,57 @@ public class SmileID {
                                                   jobId: String = "job-\(UUID().uuidString)",
                                                   allowAgentMode: Bool = false,
                                                   showAttribution: Bool = true,
+                                                  showInstruction: Bool = true,
                                                   delegate: SmartSelfieResultDelegate)
-    -> SmartSelfieInstructionsView {
+        -> some View {
         let viewModel = SelfieCaptureViewModel(userId: userId,
                                                jobId: jobId,
                                                isEnroll: true,
                                                allowsAgentMode: allowAgentMode,
                                                showAttribution: showAttribution)
-        return SmartSelfieInstructionsView(viewModel: viewModel, delegate: delegate)
+        let destination: NavigationDestination = showInstruction ?
+            .selfieInstructionScreen(selfieCaptureViewModel: viewModel, delegate: delegate) :
+            .selfieCaptureScreen(selfieCaptureViewModel: viewModel, delegate: delegate)
+        return SmileView(initialDestination: destination).environmentObject(navigationState)
     }
 
-    public class func documentVerificationScreen(userId _: String = "user-\(UUID().uuidString)",
-                                                 jobId _: String = "job-\(UUID().uuidString)",
+    public class func documentVerificationScreen(userId: String = "user-\(UUID().uuidString)",
```

Comment:
Can you add docs for this explaining the parameters? i.e. https://github.com/smileidentity/android/blob/f6a770e499105435d2c70b824ff2c63e8079f489/lib/src/main/java/com/smileidentity/compose/SmileIDExt.kt#L119